### ISSUE TITLE
Friday Hack: impact metrics

### ISSUE THREAD

**Issue by ccerv1** [opensource-observer/oso#1093]
Prework:
- Follow the [getting started](https://docs.opensource.observer/docs/get-started/#login-to-bigquery) guide to get into BigQuery from a personal google account
- Make sure you can get your Colab notebook to [run a simple query](https://docs.opensource.observer/docs/integrate/data-science#using-google-colab)

Impact Metric hack:
- Read the Request for Impact Metrics ([Project Health](https://docs.opensource.observer/blog/request-for-impact-metrics#project-health) section)
- Choose a metric you want to hack on
- Fork one of the [starter notebooks](https://drive.google.com/drive/folders/1I4exiLfZYgPwIGBEzuAMeGtJUhtS_DE7?usp=drive_link)
- Move a copy of your finished notebook to the [hacks shared drive](https://drive.google.com/drive/folders/1M8Tiz7BbmKqE7YKXrmg4SNO1TG3RisaK?usp=drive_link)
- Show your work on our [discussions thread](https://github.com/opensource-observer/oso/discussions/1090)

**Comment by ryscheng** [comment]
I posted

https://github.com/opensource-observer/oso/discussions/1090

**Comment by ccerv1** [comment]
I posted as well: https://github.com/opensource-observer/oso/discussions/1090#discussioncomment-8895919

**Comment by ccerv1** [comment]
@ravenac95 Please post your work when you have a chance and we can close this out

**Comment by ravenac95** [comment]
Ah shoot. I still then need to make something postable. I attempted to glean much more information for commits. I did successfully get #1122 but I also started looking for additional data on a variety of dimensions. These are the things I tried:

* Additional information from external pull requests. This led me to make: #1121 
* Get richer commit information from rebased commits (this ended up being a rabbit hole and probably not that useful). 

I will put something together with some of the new pull request related data!
